neqo-transport/src/connection/mod.rs

```diff
@@ -225,6 +225,69 @@ impl AddressValidationInfo {
     }
 }
 
+#[derive(Clone)]
```

Comment:
A little bit of documentation on this struct would be very helpful.  It shouldn't need to be very detailed.